### PR TITLE
Merge current Typelevel version as of 2022-07-18

### PR DESCRIPTION
**Pull request by rossabaker**
This is looking good!

I recently put a lot of work into CI, and we've had a couple posts since this project started.  This PR merges the latest development branch on the upstream site with the 47 Degrees reboot.  Feel free to discard whatever doesn't fit, but this updates the content for everybody, and helps a non-Rubyist like me get it running locally!

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -61,6 +61,7 @@ GEM
     unicode-display_width (1.8.0)
 
 PLATFORMS
+  x86_64-darwin-17
```
Comment (rossabaker) [inline Gemfile.lock:64]: I don't remember Ruby very well, and am not sure why this was necessary.  I was asked to run `bundle lock --add-platform x86_64-darwin-17`, so I did.

File: Gemfile.lock [hunk 2]
```diff
@@ -69,4 +70,4 @@ DEPENDENCIES
   jekyll-paginate
 
 BUNDLED WITH
-   2.2.32
+   2.2.24
```
Comment (rossabaker) [inline Gemfile.lock:73]: I think I could make this (and perhaps the previous) error go away if my bundler were 2.2.32.  If this causes any problems, we'll revert, and I'll figure out how to upgrade.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
+typelevel website
```
Comment (rossabaker) [inline README.md:1]: We added a lot of instructions to the , and continue to do so.  Some of them are no longer relevant, but I don't want to lose it.

File: flake.lock [hunk 4]
```diff
@@ -0,0 +1,59 @@
+{
```
Comment (rossabaker) [inline flake.lock:1]: This is a generated file.  Anyone who doesn't know what this is can comfortably ignore it.

File: flake.nix [hunk 5]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (rossabaker) [inline flake.nix:1]: This is for our dev shell and CI.  Anyone who doesn't know what this is can ignore it.

File: gemset.nix [hunk 6]
```diff
@@ -0,0 +1,306 @@
+{
```
Comment (rossabaker) [inline gemset.nix:1]: This is generated from the Gemfile lock to support the rest of the build.  Anyone who doesn't know what this is can ignore it.

File: img/ico-discord.svg [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<svg width="71" height="55" viewBox="0 0 71 55" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (rossabaker) [inline img/ico-discord.svg:1]: This icon was added when someone updated the chat link.  I don't see references to the other icons from the old site, but the files are still here, so I kept it on the merge.

File: .github/workflows/ci.yml [hunk 8]
```diff
@@ -0,0 +1,23 @@
+name: nix-flake-check
```
Comment (rossabaker) [inline .github/workflows/ci.yml:1]: I don't think the designers at 47 need to worry about this file.

**Comment by rossabaker** [comment]
I'm trying to keep the current site in sync with this prototype, but I'm not the most skilled person to get it over the finish line.  Do we have anyone from 47 Degrees or Typelevel who can take the wheel here?

**Comment by raulraja** [comment]
@israelperezglez, can you do a design pass over this PR when you have a chance? There are still a few issues related to the TL blog posts not having images. We may have to create default placeholders based on tags or suggest ideas for existing blog posts with no image. Thanks!

**Comment by rossabaker** [comment]
I expect this still runs locally with the standard jekyll commands.  For anyone without a Ruby environment, but with Nix:

```console
$ nix develop
🔨 Welcome to typelevel-org-shell

[general commands]

  jekyll     - a jekyll bundled with this site's dependencies
  menu       - prints this menu
  tl-preview - preview the Jekyll site
```

Then `tl-preview` will run the site and start a local server.

**Comment by rossabaker** [comment]
Looking at this again, there are default images here.  It looks like there's one for untagged posts, and another one shared by things tagged "technical" or "social".  I guess I could categorize our existing posts with those, if everyone agrees on the tags.

**Comment by rossabaker** [comment]
I introduced "governance" and "summits" alongside "technical" and "social" and arbitrarily put all posts into one of four categories.

I see the ones with a different image explicitly have "discord-migration.jpg"... whether they related to Discord or not.  So it's obvious how to change the pictures.  We just don't have appropriate pictures for most of these posts.

**Review by israelperezglez — APPROVED** [review]

**Pull request merged**